maintainers/maintainer-list.nix

```diff
@@ -10698,4 +10698,10 @@
     github = "felixscheinost";
     githubId = 31761492;
   };
+  gytis-ivaskevicius = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.